docs/plugindev/README.md

```diff
@@ -231,6 +231,17 @@ well.
 
 Edit the file as appropriate.
 
+### Debugging
+
+By default a plugin is built without debug symbols to reduce its binary size;
+this has the side-effect of preventing the use of a debugger towards the built
+binary.  However, when using an IDE, a different binary is used, one built by
+the IDE, and therefore the debugger can be used directly from the IDE.
+
+If you require using a debugger directly on a plugin binary, you have to build
+a binary that includes the debug symbols.  You can build such a binary by using
+`PLUGIN_ENABLE_DEBUG=1` along with your build command.
+
```

Comment:
Given that the builder now has opinions on the gcflags (to not enable function inlining), it would be good to disclose them in this md doc (not sure if this is the best section to put under though). We should
- provide instructions on how to override the inline disabling behavior
- mention that if PLUGIN_GO_FLAGS is used to tweak go other compilation flags, the disabling of function inlining could be lost unless some flags are included in the value